### PR TITLE
fix taint master example

### PR DESCRIPTION
**Pull request by sebastianliu**
change the example of taint master from `node-role.kubernetes.io/master:NoSchedule` to `node-role.kubernetes.io/master="":NoSchedule`

### CODE REVIEW

**Comment by YoungLiu** [comment]
Thanks for your reminder. I have finished to sign the CLA.

File: docs/design/design_v1.10.md [hunk 1]
```diff
@@ -315,7 +315,7 @@ Please note that
 As soon as the control plane is available, kubeadm executes following actions:
 
 - Label  the master with `node-role.kubernetes.io/master=""`
-- Taints  the master with `node-role.kubernetes.io/master:NoSchedule`
+- Taints  the master with `node-role.kubernetes.io/master="":NoSchedule`
```
[the remark this answers is not in the document]
Comment (sebastianliu) [reply docs/design/design_v1.10.md]: Firstly, thanks for your reply and review. May know why it didn't do the right thing here? After kubeadm create the control plane, we need to taints the master nodes which don't need to run application pod. Do I understand this correctly？

**Comment by timothysc** [comment]
so I double checked fresh installs and it's doing the proper thing and the documentation is correct. 

`Taints:             node-role.kubernetes.io/master:NoSchedule`

There are no additional quotes.

**Pull request closed without merging**